halpybot/commands/forcejoin.py

```diff
@@ -41,7 +41,7 @@ async def cmd_sajoin(ctx: Context, args: List[str]):
     except AttributeError:
         return await ctx.reply(f"User {args[0]} doesn't appear to exist...")
 
-    if args[1] not in config["Force join command"]["joinable"].casefold():
+    if args[1].casefold() not in config.forced_join.joinable:
```

Comment:
Pydle mandates case sensitivity for channel names. Therefore, casefold() is out of order.